go/logictest/resultparser.go

```diff
@@ -140,6 +143,10 @@ func parseLogEntry(scanner *parser.LineScanner) (*ResultLogEntry, error) {
 			eoq := strings.Index(line[colonIdx2+1:], "not ok: ") + colonIdx2 + 1
 			entry.Query = line[colonIdx2+2 : eoq-1]
 			entry.ErrorMessage = line[eoq+len("not ok: "):]
+		case Timeout:
+			eoq := strings.Index(line[colonIdx2+1:], "timeout: ") + colonIdx2 + 1
+			entry.Query = line[colonIdx2+2 : eoq-1]
+			entry.ErrorMessage = line[eoq+len("timeout: "):]
```

Comment:
Probably shouldn't have an error message for timeout cases

Reply:
Why not? Seems like a timeout is equivalent to a failure.

Reply:
That's how I view a timeout (especially an hour long timeout), as an error...

Reply:
There's no additional information in the error message. Your implementation logs the same message for all timeouts.